alephclient/cli.py

```diff
@@ -237,6 +237,19 @@ def read_json_stream(stream):
         raise click.Abort()
 
 
+@cli.command("delete-entity")
+@click.argument("entity_id", required=True)
```

Comment:
Can we narrow the type to an `int` here? (Just an assumption that these are db ids)

```suggestion
@click.argument("entity_id", required=True, type=click.INT)
```

Reply:
These are entity ids, as in the hash id of the entity that is generated from make_id. It is what you'd see in the URL if you were looking at an entity:

https://aleph.occrp.org/entities/a1da234cb12b63557fe3dd15b66e1e1608760a6e.9e770637151a4acc9245c215d78b4ac5440f4b18#mode=Employment%3Aemployee

Reply:
Ah I see. Well then ... no :)